doc/src/manual/methods.md

```diff
@@ -614,7 +614,7 @@ Start some other operations that use `f(x)`:
 julia> g(x) = f(x)
 g (generic function with 1 method)
 
-julia> t = Threads.@spawn f(wait()); yield();
+julia> t = @async f(wait()); yield();
```

Comment:
These were changed to `@spawn` in https://github.com/JuliaLang/julia/pull/55315 but I think they make sense as `@async` otherwise more orchestration is needed when multiple threads are available to make the examples work as expected.